packages/SwingSet/src/controller/upgradeSwingset.js

```diff
@@ -10,13 +10,38 @@ import {
 import { enumeratePrefixedKeys } from '../kernel/state/storageHelper.js';
 
 /**
- * @import {RunQueueEvent} from '../types-internal.js';
+ * @import {ReapDirtThreshold, RunQueueEvent} from '../types-internal.js';
  */
 
-const upgradeVatV0toV1 = (kvStore, defaultReapDirtThreshold, vatID) => {
-  // This is called, once per vat, when upgradeSwingset migrates from
-  // v0 to v1
+/**
+ * Parse a string of decimal digits into a number.
+ *
+ * @param {string} digits
+ * @param {string} label
+ * @returns {number}
+ */
+const mustParseInt = (digits, label) => {
+  assert(
+    digits.match(/^\d+$/),
+    `expected ${label}=${digits} to be a decimal integer`,
+  );
+  return Number(digits);
+};
 
+/**
+ * Called for each vat when upgradeSwingset migrates from v0 to v1.
+ *
+ * @param {KVStore} kvStore
+ * @param {(key: string) => string} getRequired
+ * @param {ReapDirtThreshold} defaultReapDirtThreshold
+ * @param {string} vatID
+ */
+const upgradeVatV0toV1 = (
+  kvStore,
+  getRequired,
```

Comment:
I still think we should change the signature of `getRequired` to be powerless and accept a `kvStore as argument. I don't like that we have multiple args that give you access to the kvStore. Probably out of scope for this PR however.